python/gui/auto_generated/qgsprevieweffect.sip.in

```diff
@@ -23,10 +23,11 @@ color blindness modes.
   public:
     enum PreviewMode
     {
-      PreviewGrayscale,
```

Comment:
While the new names are better, unfortunately we can't change these as it breaks the stable API. Can you revert the name changes please?

Reply:
Of course, wasn't even thinking about that.
Is it preferable for me to do this with a second commit, or should I update this commit and force-push to the branch for a cleaner history?

Reply:
I'd suggest multiple commits as it makes reviews easier. We can always squash on merge to get a clean history.

Reply:
Better flip the mono/grayscale order back too, just in case a script is using the raw int representations of this enum. Sorry for the hassle -- the stable API restriction can be a real pain!

Reply:
My fault for changing everything around for my first PR, especially since I remember observing how much this came up when going from 2 to 3 -- I will be more mindful going forward! I think I've got everything back, though I did retain the modified menu text if that's okay, which I think is more descriptive.

Reply:
All good, thanks!